doc/prepare_raspberrypi.rst

```diff
@@ -291,3 +291,26 @@ And to turn it off::
 Note that you need root privileges to control the LEDs.
 
 Similarly for the red (power) LED, which is called ``led1``.
+
+
+Advanced users only: Control Linux real-time properties
+-------------------------------------------------------
+Add this to the first (and only) line in ``/boot/cmdline.txt``::
```

Comment:
Should maybe explain what this does. (e.g., prevents Linux from scheduling any process on CPU core 2 except those specified with taskset).

Reply:
Added
```
See the page on Linux timing in this documentation for an introduction to
the subject.
```
Done